roles/confluent.test/molecule/certs-create.sh

```diff
@@ -62,26 +62,40 @@ for line in `sed '/^$/d' $filename`; do
       # EXT="SAN=dns:$internal"
       EXT="SAN=dns:$internal,dns:$fqdn"
 
+      FORMAT=$1
+
+      source=""
+      samples=("")
+      if [ $1 = "country" ]; then
+         source="country"
+         samples="US Canada Mexico..."
+      else
+        echo "try again"
+      fi
```

Comment:
```suggestion
```

whoops that should not have made it into the script, that was some sample code i found off the internet

Reply:
Ah ok - I will remove it then :)